app/src/components/AddPodcastModal.js

```diff
@@ -25,14 +25,17 @@ class AddPodcastModal extends React.Component {
 		this.resetModal = this.resetModal.bind(this);
 		this.submitPodcastSelections = this.submitPodcastSelections.bind(this);
 	}
+
```

Comment:
go ahead and revert all changes to this file - isn't really part of this PR, can be changed in a separate cleanup PR